MAUI/Busy-Indicator/IndicatorColor.md

```diff
@@ -0,0 +1,197 @@
+                                  IsRunning="True"
+                                  AnimationType="CircularMaterial"
+                                  Title="Loading..."
+                                  IndicatorColor="Red"/>           
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.Core;
+    using Microsoft.Maui;
+    using Microsoft.Maui.Controls;
+
+    namespace BusyIndicatorSample
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                SfBusyIndicator busyIndicator = new SfBusyIndicator()
+                {
+                    IsRunning = true,
+                    AnimationType = AnimationType.CircularMaterial,
+                    Title = "Loading...",
+                    IndicatorColor = Colors.Red
+                };
+                this.Content = busyIndicator;
+            }
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+
+## Overlay Background in SfBusyIndicator
+
+SfBusyIndicator provides option to set the overlay background. This can be done using `OverlayFill` property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:core="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
+                x:Class="BusyIndicatorSample.MainPage">
+
+        <ContentPage.Content>
+            <core:SfBusyIndicator x:Name="busyindicator"
+                                  IsRunning="True"
+                                  AnimationType="CircularMaterial"
+                                  Title="Loading..."
+                                  IndicatorColor="Red"
+                                  OverlayFill = "DarkViolet""/>           
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.Core;
+    using Microsoft.Maui;
+    using Microsoft.Maui.Controls;
+
+    namespace BusyIndicatorSample
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                SfBusyIndicator busyIndicator = new SfBusyIndicator()
+                {
+                    IsRunning = true,
+                    AnimationType = AnimationType.CircularMaterial,
+                    Title = "Loading...",
+                    IndicatorColor = Colors.Red,
+                    OverlayFill = Colors.DarkViolet
+                };
+                this.Content = busyIndicator;
+            }
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+The `OverlayFill` property type is `Brush`, so can able set Gradients as background.
```

Comment:
The `OverlayFill` property type is `Brush,` so you can set Gradients as background.